db/migrate_cards/20151117121317_migrate_logo_to_image.rb

```diff
@@ -0,0 +1,50 @@
+# -*- encoding : utf-8 -*-
+
+class MigrateLogoToImage < Card::Migration
+  def up
+    # delete cards with logo and image
+    # change name from +logo to +image
+    # update some structure cards
+
+    company_image_structure_name = 'Company+image+*type plus right+*structure'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.